enhancements/single-node/relocatable-single-node-openshift.md

```diff
@@ -0,0 +1,395 @@
+* Simplify SNO deployment at the far edge: Non-technical operators should be able to reconfigure SNO at deployment time.
+
+____________
+
+
+### Non-Goals
+
+* Relocatable multi-node clusters.
+* 
+
+## Proposal
+
+### Configuring single node OpenShift to be reloactable
+
+A few configurations will be applied before the node is relocated to allow the networking changes and reconfiguration required at the edge deployment.
+These changes will be applied using a MachineConfig that will:
+1. Create an additional known IP on the br-ex bridge. 
+2. Disable kubelet service (to prevent kubelet from starting upon deployment at the edge).
+3. Create systemd service to re-configure the SNO. 
+
+#### IP address modification
+
+Currently, when a single node OpenShift boots, ovs-configuration.service creates the br-ex bridge with the node’s default interface and sets the node’s IP as the environment variable for kubelet.service and crio.service.
+A relocatable SNO will likely use different network settings at the factory where it is first installed and at the edge site where it is deployed.
+To accommodate these network changes, we will decouple the host’s physical interface IP address from the internal node IP used by crio and kubelet.
+During the initial installation at the factory, a MachineConfig will be applied that adds a NetworkManager configuration script that adds an IP to the br-ex bridge when created by ovs-configuration.service.
+This IP will be the same IP as ovs-configuration.service configured on the br-ex bridge during the installation. 
+The MachineConfig will also add an env override to kubelet.service and crio.service that sets the kubelet NODE_IP to the internal node IP added by the NetworkManager pre-up script.
+See example MachineConfig [here](https://github.com/eranco74/image-based-installation-poc/blob/master/bake/node-ip.yaml).
+This decoupling should allow OCP to keep running as if the hostIP didn't change.
+
+[//]: # (check the implication of this change on diffrent traffic cases, what IP will be the endpoint IP services will get?, what about ingress traffic and nodePort?)
+#### Certificate expiry
+
+When deploying SNO, the kubelet and node-bootstrapper certificate signing requests need to get approved and issued for kubelet to authenticate with the kube-apiserver and for the node to register.
+The initial certificate is currently valid for 24 hours. Extending this certificate’s validity to 30 days will reduce the time needed to complete the reconfiguration process at the edge.
+
+#### SNO reconfiguration
+We will add a new systemd service to reconfigure the node upon deployment at the edge site (see [Alternatives for SNO reconfiguration](#Alternatives for SNO reconfiguration)).
+This new service will read the site-specific configuration (site-config) and reconfigure the following:
```

Comment:
Just a note on naming.
Perhaps don't use site-config unless you're actually using the `SiteConfig` resource already created for deploying clusters using ZTP

ref: https://docs.openshift.com/container-platform/4.13/scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.html#ztp-deploying-a-site_ztp-deploying-far-edge-sites